src/Agent.Worker/RetryHelper.cs

```diff
@@ -1,15 +1,27 @@
 ﻿using System;
 using System.Threading.Tasks;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
 
 namespace Microsoft.VisualStudio.Services.Agent.Worker
 {
     internal class RetryHelper
     {
+        /// <summary>
+        /// Returns exponential delay - depending on number of retry
+        /// </summary>
+        /// <returns></returns>
+        public static int ExponentialDelay(int retryNumber)
+        {
+            return (int)(Math.Pow(retryNumber, 2) * 5);
```

Comment:
The `Delay` method takes a value in milliseconds it means that the current timings for timeouts are very low. 
Could you please fix the calculation of timeouts?

```CSharp
public static Task Delay(int millisecondsDelay, CancellationToken cancellationToken);
```

Reply:
Thanks! Fixed - added comments also for this functor and retry method.